test/e2e/storage/framework/volume_resource.go

```diff
@@ -204,7 +204,7 @@ func (r *VolumeResource) CleanupResource() error {
 				}
 
 				if pv != nil {
-					err = e2epv.WaitForPersistentVolumeDeleted(f.ClientSet, pv.Name, 5*time.Second, 5*time.Minute)
+					err = e2epv.WaitForPersistentVolumeDeleted(f.ClientSet, pv.Name, 5*time.Second, f.Timeouts.PVDelete)
```

Comment:
I think that the timeouts should be using the defined in the framework, but sig-storage should check if it is ok to reduce it here

```
test/e2e/framework/timeouts.go: pvDeleteTimeout                  = 3 * time.Minute
test/e2e/framework/timeouts.go: pvDeleteSlowTimeout              = 20 * time.Minute
test/e2e/framework/timeouts.go:         PVDelete:                  pvDeleteTimeout,
test/e2e/framework/timeouts.go:         PVDeleteSlow:              pvDeleteSlowTimeout,
```

Reply:
It's hard to tell whether this will work in all cases and with all drivers.

Can we bump up pvDeleteTimeout to 5 minutes as part of this PR, just to be on the safe side?

Reply:
Done